hub/migrations/0012_alter_constance_config.py

```diff
@@ -0,0 +1,44 @@
+import json
+
+from constance import config
+from django.db import migrations, models
+
+
+def alter_constance_config(apps, schema_editor):
+    # Check that the USER_METADATA_FIELDS has `full_name` set
+    user_metadata = json.loads(getattr(config, 'USER_METADATA_FIELDS'))
+    name_set = False
+    for field in user_metadata:
+        if field['name'] == 'full_name':
+            name_set = True
+    if not name_set:
+        user_metadata.append({
```

Comment:
To be consistent with default value and because `full_name` comes first, I would use `insert(0, ...)` instead of `append(...`.